pkg/flowaggregator/flowaggregator.go

```diff
@@ -724,3 +818,130 @@ func (fa *flowAggregator) createInfoElementForTemplateSet(ieName string, enterpr
 	}
 	return ie, nil
 }
+
+// updateIfConfigChanges has a watcher to detect if modification happens. Then check whether we should update clickhouse or flow-collector at runtime
+func (fa *flowAggregator) updateIfConfigChanges(stopCh <-chan struct{}) {
+	if fa.configData == nil {
+		var err error
+		fa.configData, err = ioutil.ReadFile(fa.configFile)
+		if err != nil {
+			klog.ErrorS(err, "Get flow aggregator configData failed:")
+		}
```

Comment:
why don't we return the error to kill the FA process? This is a serious error

Reply:
fixed.